examples/OrderEdit/OrderEditAddStagedActionAction.json

```diff
@@ -0,0 +1,6 @@
+{
+  "action": "addStagedAction",
+  "stagedAction": [
+    "stagedOrderUpdateAction"
```

Comment:
Please either change to 
```
"stagedAction": [
]
```
or one valid staged example action e.g.:
```
"stagedAction": [
  {
     "action": "setComment",
     "comment": "commentString"
  }
]
```

Reply:
example added

Reply:
I'm so sorry. `setComment` is no stageable action. Could you please change it so some action with can be staged?